src/code/IFindPSResource.cs

```diff
@@ -28,32 +28,39 @@ public interface IFindPSResource
     /// Name: supports wildcards
     /// Type: Module, Script, Command, DSCResource (can take multiple)
     /// Examples: Search -Type Module -Repository PSGallery
-    ///           Search "Az*" -Type Module -Repository PSGallery
-    /// - No prerelease: http://www.powershellgallery.com/api/v2/Search()?$filter=IsLatestVersion&searchTerm='tag:Module'
-    /// - Include prerelease: http://www.powershellgallery.com/api/v2/Search()?$filter=IsAbsoluteLatestVersion&searchTerm='tag:Module'&includePrerelease=true
+    ///           Search -Type Module -Name "Az*" -Repository PSGallery
+    /// TODO: discuss consolidating Modules and Scripts endpoints (move scripts to modules endpoint)
+    /// TODO Note: searchTerm is tokenized by whitespace.
+    /// - No prerelease: http://www.powershellgallery.com/api/v2/Search()?$filter=IsLatestVersion&searchTerm='Az* tag:PSModule'
+    /// - Include prerelease: http://www.powershellgallery.com/api/v2/Search()?$filter=IsLatestVersion&searchTerm='az* tag:PSScript'&includePrerelease=true
     /// </summary>
     PSResourceInfo FindTypes(ResourceType packageResourceType, string packageName, PSRepositoryInfo repository, bool includePrerelease);
 
     /// <summary>
     /// Find method which allows for searching for command names and returns latest version of matching packages.
-    /// Name: supports wildcards
+    /// Name: no wildcard support
     /// Examples: Search -Name "Command1", "Command2" -Repository PSGallery
+    /// - No prerelease: http://www.powershellgallery.com/api/v2/Search()?$filter=IsLatestVersion&searchTerm='Az* tag:PSCommand_Command1 tag:PSCommand_Command2'
+    /// - Include prerelease: http://www.powershellgallery.com/api/v2/Search()?$filter=IsLatestVersion&searchTerm='Az* tag:PSCommand_Command1 tag:PSCommand_Command2'&includePrerelease=true
     /// </summary>
     PSResourceInfo FindCommandName(string[] commandNames, PSRepositoryInfo repository, bool includePrerelease);
 
     /// <summary>
     /// Find method which allows for searching for DSC Resource names and returns latest version of matching packages.
-    /// Name: supports wildcards
+    /// Name: no wildcard support
```

Comment:
With this API call, we can support wildcards. So do we want to revert this change back?
```suggestion
    /// Name: Support wildcards.
```